lib/awful/widget/layoutlist.lua

```diff
@@ -0,0 +1,442 @@
+----------------------------------------------------------------------------
+--- Display the available client layouts for a screen.
+--
+-- This is what the layoutlist looks like by default with a vertical layout:
+--
+--@DOC_awful_widget_layoutlist_default_EXAMPLE@
+--
+-- In the second example, it is shown how to create a popup in centered in
+-- the screen:
+--
+--@DOC_awful_widget_layoutlist_popup_EXAMPLE@
+--
+-- This example show how to extend the `awful.widget.layoutbox` to show a layout
```

Comment:
`This example extends 'awful.widget.layoutbox' to show a layout list popup:`
or just change `show` to `shows`

Reply:
Done